### ISSUE TITLE
`misc-unused-using-decls` false positive on exported usings

### ISSUE THREAD

**Issue by RoqueDeicide** [llvm/llvm-project#162619]
I'm exporting a using from a module:

```cpp
export using ::IDirectMusic;
```
`clang-tidy` incorrectly reports this as an unused `using` declaration.

## Version
`21.1.2`
